.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -50,6 +50,9 @@ body:
         - Documentation (part:docs)
         - Unit, integration and performance tests (part:tests)
         - Build script, CI, dependencies, etc. (part:tooling)
+        # TODO(cookiecutter): Add other parts
+        # Please have in mind that that the part:xxx labels need to
+        # be created in the GitHub repository.
```

Comment:
General comments about all TODOs added here: You should probably address them and remove them. Since this is a preexisting repository, it is very likely you want to remove them without much thought, as the setup should have been already done.

We should have a way to determine if this is a first time generation or not and only put `TODOs` for first time generation, but for now it needs to be manually cleaned up on every update :disappointed: 

* https://github.com/frequenz-floss/frequenz-repo-config-python/issues/186

Reply:
I recommended to leave them in since they will come back anyway :) But I realize they could cause problems with nox, right?